test_harness/main.py

```diff
@@ -93,6 +133,20 @@ def cli():
         help="Path to a file of tests to be run. This would be the same output from downloading the tests via `download_tests()`",
     )
 
+    parser.add_argument(
```

Comment:
I think these should have a default value and be required. Either that or include what was passed in the error message inside the graph validation.